### PR TITLE
Add Doris error code table

### PR DESCRIPTION
**Pull request by hf200012**
Add Doris error code table

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/administrator-guide/doris_error_code.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+under the License.
+-->
+
+# Doris错误代码表
+
+| 错误码 | 错误信息                                                     |
+| :----- | :----------------------------------------------------------- |
+| 1005   | 创建表格失败，在返回错误信息中给出具体原因                   |
+| 1007   | 数据库已经存在，不能创建同名的数据库                         |
+| 1008   | 数据存不存在，无法删除                                       |
```
Comment (xy720) [inline docs/zh-CN/administrator-guide/doris_error_code.md:34]:
```suggestion
| 1008   | 数据库不存在，无法删除                                       |
```

**Review by yangzhg — COMMENTED** [review]
Please add an English Version

File: docs/zh-CN/administrator-guide/doris_error_code.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+| 1372   | 密码长度不够                                                 |
+| 1396   | 用户执行的操作运行失败                                       |
+| 1471   | 指定表不允许插入数据                                         |
+| 1507   | 删除不存在的分区，且没有指定如果存在才删除的条件             |
+| 1508   | 无法删除所有分区，请改用DROP TABLE                           |
+| 1517   | 出现了重复的分区名字                                         |
+| 1567   | 分区的名字不正确                                             |
+| 1621   | 指定的系统变量是只读的                                       |
+| 1735   | 表中不存在指定的分区名                                       |
+| 1748   | 不能将数据插入具有空分区的表中使用“ SHOW PARTITIONS FROM  tbl”来查看此表的当前分区 |
```
Comment (xy720) [inline docs/zh-CN/administrator-guide/doris_error_code.md:89]:
```suggestion
| 1748   | 不能将数据插入具有空分区的表中。使用“ SHOW PARTITIONS FROM  tbl”来查看此表的当前分区 |
```

**Comment by hf200012** [comment]
ok

**Review by xy720 — CHANGES_REQUESTED** [review]

**Comment by hf200012** [comment]
> Please add an English Version

ok

**Review by yangzhg — DISMISSED** [review]
LGTM

**Review by morningman — COMMENTED** [review]
need to add it to he side bar

File: docs/zh-CN/administrator-guide/doris_error_code.md (hunk 1 above)
Comment (hf200012) [reply docs/zh-CN/administrator-guide/doris_error_code.md]: solved

File: docs/zh-CN/administrator-guide/doris_error_code.md (hunk 2 above)
Comment (hf200012) [reply docs/zh-CN/administrator-guide/doris_error_code.md]: solved

**Comment by hf200012** [comment]
> need to add it to he side bar

solved

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**